frontend/src/pages/SignIn/SignIn.tsx

```diff
@@ -60,10 +62,12 @@ const SignIn = ({
               <ItemsBox component="form" onSubmit={handleSubmit}>
                 <GridWrapper container spacing={1}>
                   <GridSignInput
+                    onClick={(event) => getFieldName(event)}
```

Comment:
No need to use arrow func here:
```suggestion
                    onClick={getFieldName}
```